core/src/main/scala/zio/config/ConfigDescriptor.scala

```diff
@@ -134,38 +136,38 @@ object ConfigDescriptor {
     g: B => Either[String, A]
   ) extends ConfigDescriptor[K, V, B]
 
-  def bigDecimal(path: String): ConfigDescriptor[String, String, BigDecimal] =
-    ConfigDescriptor.Source(path, ConfigSource.empty, PropertyType.BigDecimalType) ?? "value of type bigdecimal"
+  val bigDecimal: ConfigDescriptor[String, String, BigDecimal] =
+    ConfigDescriptor.Source(ConfigSource.empty[String, String], PropertyType.BigDecimalType) ?? "value of type bigdecimal"
```

Comment:
I hope `ConfigSource.empty` works?

Reply:
It works, but IDEA is complaining. I'll revert it.